src/main/java/com/woowacourse/pelotonbackend/common/exception/DuplicateReportException.java

```diff
@@ -0,0 +1,12 @@
+package com.woowacourse.pelotonbackend.common.exception;
+
+import com.woowacourse.pelotonbackend.common.ErrorCode;
+import lombok.Getter;
+
+@Getter
+public class DuplicateReportException extends DuplicateException {
```

Comment:
Exception 이름에도 엔티티 명이 먼저오면 더 쉽게 읽어질 것 같아요
```suggestion
public class ReportDuplicateException extends DuplicateException {
```